pieforproviders/pieforproviders
# Change Schedules from start_time/end_time to duration

## 💅🏼 What issue does this fix?



Partial PR for #1717 - this contains the first two check boxes from [this comment](https://github.com/pieforproviders/pieforproviders/issues/1717#issuecomment-948088253), moving schedules from start/end time to duration

## 🍂 Before You Submit


## 🛷 Deployment


## 🧵 Steps to set up locally

```
bundle exec rails db:migrate
```

## 🧳 Steps to test

- Create a new child with no Schedule information
  - They should be created with an 8 hour duration for all 5 weekdays
- Create an attendance with no checkout on a child's scheduled day
  - Its duration should be the same as the scheduled day
